### PR TITLE
DS-206 - missing svg icons

### PR DESCRIPTION
**Pull request by marcinsawicki**
https://livechatinc.atlassian.net/browse/DS-206

### CODE REVIEW

File: packages/icons/svg/material/camera.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (AlithAnar) [inline packages/icons/svg/material/camera.svg:1]: do we need explicit dimensions inside svgs? in AA we are removing them from svgs in favor of upper-level size management

Comment (marcinsawicki) [reply packages/icons/svg/material/camera.svg:1]: I see that every single SVG file has those dimensions, which do not cause any problems. Do You think that we should change that?

Comment (AlithAnar) [reply packages/icons/svg/material/camera.svg:1]: nope, I was just curious :>

Comment (MichalPaszowski) [reply packages/icons/svg/material/camera.svg:1]: I think these are removed when converted to React components.

**Review by MichalPaszowski — APPROVED** [review]

**Pull request merged**